app/assets/javascripts/discourse/app/controllers/bookmark.js

```diff
@@ -89,6 +90,12 @@ export default Controller.extend(ModalFunctionality, {
     if (this._editingExistingBookmark()) {
       this._initializeExistingBookmarkData();
     }
+
+    scheduleOnce("afterRender", () => {
```

Comment:
scheduleOnce, with an anonymous function doesn't do anything, context will be different each time, so it will be recreated.

I would just go with schedule here.